### ISSUE TITLE
add CLI deployment to each framework

### ISSUE THREAD

**Issue by edublancas** [ploomber/doc#102]
once https://github.com/ploomber/cli/issues/36 is merged, we can add a command for users to deploy an application from the CLI

for example, in flask:

```sh
pip install ploomber-cloud
ploomber-cloud key YOUR-KEY
ploomber-cloud examples docker/flask
ploomber-cloud init
ploomber-cloud deploy --watch
```

**Comment by edublancas** [comment]
let's start with the ones that appear in our grid: voila, panel, solara, streamlit, and shiny R

note that some examples require OpenAI keys, so let's choose some example that doesn't require anything but still shows something to spark interest in users

**Comment by bryannho** [comment]
@edublancas this was addressed in #138 so I think it can be closed

**Comment by edublancas** [comment]
good catch
